test/TestUtils.ts

```diff
@@ -34,19 +41,14 @@ export function stubCatalogFixture(): void {
 	});
 }
 
-/**
- * Initial setup needed by all oclif command unit tests.
- *
- * Example:
- * setupCommandTest
- * 	.command(['scanner:run', '-t', 'test-code'])
- * 	.it('Scanner Run Relative Path Succeeds', ctx => {
- * 		expect(ctx.stdout).to.contain('No rule violations found.');
- * 	});
- */
-export const setupCommandTest = test
-	.do(() => TestOverrides.initializeTestSetup())
-	.stdout()
-	.stderr();
+export function runCommand(command: string): ExecCmdResult<AnyJson> {
```

Comment:
This is the new method that should be used for E2E tests instead of `setupCommandTest`.